File: fixtures/ternary-call.kt
fun pick(value: Double): Double {
  return value * 2.0
}

fun choose(flag: Boolean): Double {
  val chosen: Double = if (flag) pick(1.0) else pick(2.0)
  return chosen
}

fun nested(flag: Boolean, other: Boolean): Double {
  return if (flag) pick(3.0) else (if (other) pick(4.0) else 5.0)
}

File: fixtures/ternary-call.swift
func pick(value: Double) throws -> Double {
  return (value * 2.0)
}

func choose(flag: Bool) throws -> Double {
  let chosen: Double = (flag ? pick(value: 1.0) : pick(value: 2.0))
  return chosen
}

func nested(flag: Bool, other: Bool) throws -> Double {
  return (flag ? pick(value: 3.0) : (other ? pick(value: 4.0) : 5.0))
}
